docker-compose.dist.yml

```diff
@@ -7,50 +7,31 @@ services:
         container_name: koillection
         build:
             dockerfile: Dockerfile.dev
-        restart: always
         ports:
             - 81:80
         environment:
-            - APP_ENV=prod
-            - APP_DEBUG=0
-            - DB_DRIVER=pdo_pgsql
+            - APP_ENV=dev
+            - APP_DEBUG=1
+            - DB_DRIVER=pdo_mysql
             - DB_NAME=koillection
-            - DB_HOST=postgresql
-            - DB_PORT=5432
-            - DB_USER=postgres
+            - DB_HOST=mysql
+            - DB_PORT=3306
+            - DB_USER=root
             - DB_PASSWORD=password
             - DB_VERSION=15
             - PHP_TZ=Europe/Paris
-            - HTTPS_ENABLED=1
+            - HTTPS_ENABLED=0
         depends_on:
             - mysql
-            - postgresql
         volumes:
             - "./docker/volumes/koillection/public/uploads:/var/www/koillection/public/uploads"
             - ./:/var/www/koillection
 
-    postgresql:
-        container_name: postgresql
-        image: postgres:15
-        environment:
-            - POSTGRES_DB=koillection
-            - POSTGRES_USER=postgres
-            - POSTGRES_PASSWORD=password
-            - POSTGRES_HOST_AUTH_METHOD=trust
-        ports:
-            - 5432:5432
-        volumes:
-            - "./docker/volumes/postgresql:/var/lib/postgresql/data"
-
     mysql:
-        container_name: mysql
         image: mysql:latest
-        command: --default-authentication-plugin=mysql_native_password
```

Comment:
I'm not sure if this should be removed. From what I've read, missing this line may sometimes cause problems with MySQL 8.0, so please consider reverting (I tested with this line and it works, too).

Reply:
Aligned to upstream, it's not used there. No problems observed with this line removed so far.